test/common/formatter/substitution_formatter_test.cc

```diff
@@ -2680,7 +2680,7 @@ TEST(SubstitutionFormatterTest, GrpcStatusFormatterCamelStringTest) {
   GrpcStatusFormatter formatter("grpc-status", "", absl::optional<size_t>(),
                                 GrpcStatusFormatter::Format::CamelString);
   NiceMock<StreamInfo::MockStreamInfo> stream_info;
-  Http::TestRequestHeaderMapImpl request_header;
+  Http::TestRequestHeaderMapImpl request_header{{"content-type", "application/grpc+proto"}};
```

Comment:
Looking at isGrpcRequestHeaders:
https://github.com/envoyproxy/envoy/blob/5523257c7059f0748611b0d83cfa7c28268d0df9/source/common/grpc/common.cc#L55

You would also need to set a path for the request.

Reply:
Thanks a lot! You are right! @ohadvano